tests/wpunit/Tribe/DependencyTest.php

```diff
@@ -49,7 +49,7 @@ public function validClassToPueProvider() {
 
 	/**
 	 * @test
-	 * @dataProvider validClassToPueProvider
+	 * @dataProvider valid_calls_to_pue_provider
```

Comment:
Sanity check - was the witch from `Class` to `calls` intentional?

Reply:
Typo, thanks.